src/WorkOS.net/Services/SSO/Entities/Connection.cs

```diff
@@ -19,11 +20,18 @@ public class Connection
         [JsonProperty("id")]
         public string Id { get; set; }
 
+        /// <summary>
+        /// The linked state of the Connection.
+        /// </summary>
+        [JsonProperty("state")]
+        public ConnectionState State { get; set; }
+
         /// <summary>
         /// The linked status of the Connection.
         /// </summary>
         [JsonProperty("status")]
-        public ConnectionStatus Status { get; set; }
+        [ObsoleteAttribute("The status property is obsolete. Please use state instead.", false)]
```

Comment:
Could we use the same casing as the property names in these deprecation messages?

```cs
[ObsoleteAttribute("The Status property is obsolete. Please use State instead.", false)]
```